.builder/actions/mock_server_setup.py

```diff
@@ -0,0 +1,40 @@
+"""
+Setup local mock server for tests
+"""
+
+import Builder
+
+import os
+import subprocess
+
+
+class MockServerSetup(Builder.Action):
+    """
+    Set up this machine for running the mock server test
+
+    This action should be run in the 'pre_build_steps' or 'build_steps' stage.
+    """
+
+    def run(self, env):
+        self.env = env
+        if os.name == 'nt':
+            python_execute = "python"
+        else:
+            python_execute = "python3"
+
+        # install dependency for mock server
+        result = self.env.shell.exec(python_execute,
+                                     '-m', 'install', 'h11', 'trio')
```

Comment:
Use the string "{python}"
Builder will substitute the correct python path for this image